packages/glimmer-compiler/lib/template-compiler.ts

```diff
@@ -10,6 +10,10 @@ export interface CompileOptions {
   moduleName?: string;
 }
 
+function isTrustingValue(value) {
```

Comment:
This should be "isTrusted" (meaning the value has been TSA pre-✔️ ed)

trusted is TSA pre-✔️ed travelers, trusting is a lousy security guard just let everyone through